Description: Fixes the compatibility with the version of Ivy in Debian
Author: Daniel Leidert (dale) <daniel.leidert@wgdd.de>
Forwarded: no
--- a/src/main/java/net/sf/antcontrib/net/URLImportTask.java
+++ b/src/main/java/net/sf/antcontrib/net/URLImportTask.java
@@ -158,7 +158,7 @@
 			adapter = new Ivy20Adapter();
 		}
 		catch (ClassNotFoundException e) {
-			adapter = new Ivy14Adapter();
+			// adapter = new Ivy14Adapter();
 		}
 		
 		String setId = org + "." + module + "." + rev + ".fileset";
--- a/build.xml
+++ b/build.xml
@@ -109,6 +109,7 @@
 		       classpathref="compile.classpath"
 		       source="${jdk.source}"
 		       target="${jdk.target}"
+		       excludes="**/Ivy14Adapter.java"
 		  />
 		
 		<copy todir="${target.classes.dir}">
